lighthouse-cli/test/smokehouse/test-definitions/errors/error-expectations.js

```diff
@@ -31,9 +31,13 @@ const infiniteLoop = {
   artifacts: {
     PageLoadError: {code: 'PAGE_HUNG'},
     devtoolsLogs: {
+      // Fraggle Rock treats devtoolsLogs as regular artifacts which are not collected on error.
```

Comment:
Are we ok with this change? I remember this was a specific request (preserve some level of trace/devtools even in the case of errors), though it's hard to track down the exact conversation. Some of it is in #8865 (ctrl-f for "devtools")

Reply:
The fix for devtools logs is tracked by https://github.com/GoogleChrome/lighthouse/issues/11313 though I'm happy to do it here instead if you prefer.

I do not believe traces are worth it, but it's not much work to add them too.

Reply:
> The fix for devtools logs is tracked by #11313

Ah, didn't realize that. Worth an `FR-COMPAT` comment?

> though I'm happy to do it here instead if you prefer.
> 
> I do not believe traces are worth it, but it's not much work to add them too.

Haha, well as I recall I thought it was easiest to just delete them and had to be convinced of their value, so I'm not the person to ask. I assume "the primary desire and reason for providing this mostly useless LHR" (from https://github.com/GoogleChrome/lighthouse/pull/8865#issuecomment-497481383) was due to LR, but I don't remember if the reason was just the debugging case.

I mainly brought it up because this is mostly closing the door on them (wasn't aware of the #11313 entry) so double checking that no one is going to be missing them six months from now or whatever.

Reply:
Added support for devtools logs, skipping traces for now unless someone else would like to speak up about how useful debug traces have been in the past 2 years :)

@connorjclark @exterkamp @paulirish perhaps?

Reply:
could also add an entry for it in #11313 and check when @paulirish is back from OoO

Reply:
done